app/views/movie/create.scala.html

```diff
@@ -0,0 +1,42 @@
+@import helper._
+@import modules.utility.twirl.CustomHelpers._
+@import modules.movie.{CreateMovieForm}
+@import modules.util.{Country, Language}
+@(movieForm: Form[CreateMovieForm], id: Long = 0)(implicit request: MessagesRequestHeader, flash: Flash)
+
+@main("Create movie") {
+    @form(
+        if (movieForm("title").value) modules.movie.routes.MovieController.update(id) else modules.movie.routes.MovieController.saveNewMovie(),
```

Comment:
Interesting idea, but would this still work after form error if you've entered title and some other field failed? Maybe change this predicate to more stable one, for example by supplying movie ID as an argument for the template.

Reply:
I don't know what exactly do you mean but I can change that